pkg/util/fuzzystrmatch/soundex_test.go

```diff
@@ -39,12 +42,25 @@ func TestSoundex(t *testing.T) {
 		},
 		{
 			Source:   "🌞",
-			Expected: "000",
+			Expected: "",
```

Comment:
why did this one change? it looks like PG returns 000 for it (binary `d0303030`)